### PR TITLE
Lots of Collection/Genres updates for Plex agent

### PR DESCRIPTION
**Pull request by Darklyter**
Sorry about the double commit for this...  realized after pushing my changes up to my repo that I was way behind the Head, was easier to revert, merge and re-push

### CODE REVIEW

**Review by Tscherno — APPROVED** [review]
Wokrs for me!

File: 3rd party/StashPlexAgent.bundle/Contents/Code/__init__.py [hunk 1]
```diff
@@ -83,110 +71,241 @@ def search(self, results, media, lang):
...
+        allow_scrape = False
+        if (Prefs["RequireOrganized"] and data["organized"]) or not Prefs["RequireOrganized"]:
+            if DEBUG and Prefs["RequireOrganized"]:
+                Log("Passed 'Organized' Check, continuing...")
+            if (Prefs["RequireURL"] and data["url"]) or not Prefs["RequireURL"]:
+                if DEBUG and Prefs["RequireURL"]:
+                    Log("Passed 'RequireURL' Check, continuing...")
+                if (Prefs["RequireStashID"] and len(data["stash_ids"])) or not Prefs["RequireStashID"]:
+                    if DEBUG and Prefs["RequireStashID"]:
+                        Log("Passed 'RequireURL' Check, continuing...")
```
Comment (bnkai) [inline 3rd party/StashPlexAgent.bundle/Contents/Code/__init__.py:91]: typo ? `RequireURL` -> `RequireStashID`

File: 3rd party/StashPlexAgent.bundle/README.md [hunk 2]
```diff
@@ -5,14 +5,26 @@ Scenes are matched based on filename (without path or extension) against the Sta
...
 
-Also this agent only handles scenes currently.  I haven't played with movies in Stash yet, but can take a look if there is interest.  Currently the Plex ADE agent handles that for me.
\ No newline at end of file
+Also this agent only handles scenes currently.  I haven't played with movies in Stash much yet, but can take a look if there is interest (though it will optionally create Collections based on defined movies).  Currently the Plex ADE agent handles that for me.
+
+Also a bit of explanation for Sites vs Studios:
+
+I help out with TPDB, so I'm very much in the Site -> Studio -> Network mentality.  In Stash it is simply "Studio".
+
+For my thinking, a Stash studio that is directly connected to the scene is the "Site".  If that site has a parent studio, that is defined as "Studio".  If the scene studio has a grandparent, that would be "Network" (thgouh I'm not doign anything with that yet.
```
Comment (bnkai) [inline 3rd party/StashPlexAgent.bundle/README.md:26]: Typo
? `(thgouh I'm not doign anything with that yet.`  -> `(I'm not doing anything with that yet).`

File: 3rd party/StashPlexAgent.bundle/Contents/Code/__init__.py (hunk 1 above)
Comment (Darklyter) [reply 3rd party/StashPlexAgent.bundle/Contents/Code/__init__.py:91]: Yeah, in my prefs I have those two options...  guessing you can tell the order I copy/pasted.  :D

File: 3rd party/StashPlexAgent.bundle/README.md (hunk 2 above)
Comment (Darklyter) [reply 3rd party/StashPlexAgent.bundle/README.md]: Okey doke, typos fixed

**Comment by Darklyter** [comment]
Got a couple of typos corrected that bnkai found.  And this folks is why you shouldn't late night commit.  :-)

**Pull request merged**